pkg/k8s/portforward.go

```diff
@@ -138,6 +139,7 @@ func newPortForward(
 }
 
 // Run creates and runs the port-forward connection.
+// It blocks upon a successful connection; otherwise it returns an error.
```

Comment:
I'm a little unclear on the wording here.  Does this mean it blocks until the connection is destroyed?

Reply:
Yeah that's right. I'll reword the comment to make it clearer.